### ISSUE TITLE
Accessibility: Some controls/buttons missing ARIA labels

### ISSUE THREAD

**Issue by Murcielago7384** [OxygenCobalt/Auxio#1455]
### Describe the bug/crash

There are several controls/buttons in Auxio that are missing ARIA labels. These are strings of text under the hood that allow screen-readers like TalkBack to identify what the button is for.

To reproduce this, you will need TalkBack running. Both mainstream Android and Graphene OS have TalkBack available.
* If you explore Auxio with TB running, you will notice that most controls already have an ARIA label. For example, going into an album, TB will identify the shuffle button and read-aloud "shuffle".
However, there are several places where such a label does not exist, and TB only announces "button".

* Again, while inside of an album, the 'back' button the upper left corner.
* When scrolling through a list, such as artists or albums, the control next to each entry.
* These are the only ones I came across, but there may be others present.

If this is something new to the dev(s), I completely understand. I like to point people towards Mozilla, as they have very detailed documentation on ARIA labels and broader accessibility.

Thank you!

### Describe the intended behavior

All buttons/controls should have a unique ARIA label, so blind/visually impaired users who rely on a screen-reader like TalkBack can quickly and identify what each button is for.

### What Android version do you use?

Android 17

### What device model do you use?

Google Pixel 8A, running Graphene OS.

### Provide a sample file

I am unsure how to do a screen recording. But TalkBack can be enabled in "system settings > accessibility".

### Bug report

N/A

### Duplicates

- [x] I have checked the [Troubleshooting](https://github.com/OxygenCobalt/Auxio/wiki/Troubleshooting) page.
- [x] I have checked this issue for duplicates.
- [x] I have checked that this issue occurs on the [lastest version](https://github.com/OxygenCobalt/Auxio/releases).
- [x] I agree to the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by OxygenCobalt** [comment]
Yeah, I haven't been focusing on ARIA that much but I should, will look into it @Murcielago7384

**Comment by Murcielago7384** [comment]
@OxygenCobalt Most of the app is fine, it's just in a couple of places. But I would  really appreciate it whenever you have some spare time. I'm very passionate about this sort of thing! :)
